pkg/debugapi/chequebook.go

```diff
@@ -110,6 +110,12 @@ func (s *Service) chequebookLastPeerHandler(w http.ResponseWriter, r *http.Reque
 
 	var lastSentResponse *chequebookLastChequePeerResponse
 	lastSent, err := s.swap.LastSentCheque(peer)
+	if errors.Is(err, postagecontract.ErrChainDisabled) {
```

Comment:
But wait, we need to change the functions in `noOpSwap` to return this error still right?

Reply:
it's already doing that: pkg/settlement/swap/swap.go:408